slitlessutils/examples/wr96.py

```diff
@@ -10,6 +10,7 @@
 import shutil
 
 import slitlessutils as su
+from slitlessutils.core.preprocess.crrej.drizzle import drizzle
```

Comment:
Idon't think we should import like this, since SU is already imported.

Reply:
No longer applicable due to https://github.com/spacetelescope/slitlessutils/pull/27/commits/1d61459b168e134a749c021a1c6b552bf26ecb62